src/core/positioning/positioning.cpp

```diff
@@ -325,7 +325,13 @@ void Positioning::processCompassReading()
       default:
         break;
     }
+
     orientation += mCompass.reading()->azimuth();
+    if ( orientation < 0.0 )
+    {
+      orientation = 360 + orientation;
+    }
```

Comment:
```suggestion
    orientation %= 360;
```

short form that will handle even the scenario of a value > 360

Reply:
ignore this, it's different in c than python